### DISCUSSION TITLE
Override AdminController Method

### DISCUSSION THREAD

**Discussion by sly8787** [pimcore/pimcore#8121] [Q&A / Support] [answered]
Hi guys

Is there somewhere an example which shows how to override for example a method of the AdminController?

Best regards

**Comment by NiklasBr** [comment, marked as answer]
There is a chapter in the Symfony docs which may help: [How to Override any Part of a Bundle](https://symfony.com/doc/4.4/bundles/override.html):

> If the controller is a service, see the next section on how to override it. Otherwise, define a new route + controller with the same path associated to the controller you want to override (and make sure that the new route is loaded before the bundle one).
